src/routes/auth.js

```diff
@@ -161,17 +163,20 @@ export default function authRoute() {
       playstyle: req.body.playstyle,
       years: req.body.years,
       experience: req.body.experience,
+      name,
     });
 
     res.status(200).end();
   }));
 
   router.put('/scouting', authenticateJWT, checkAuthenticated, asyncHandler(async (req, res) => {
     const {
-      first_name, last_name, first_team, second_team, third_team, position, playstyle, years, experience,
+      first_name, last_name, first_team, second_team, third_team, position, playstyle, years, experience, club_uuid,
     } = await prisma.users.findUnique({ where: { uuid: req.user.uuid } });
 
-    // Email to head scout, with spplication and national team profile information.
+    const { name } = await getClub(club_uuid);
```

Comment:
See comment above about improving the context for the email, you'll also need to change the value in the payload below it

```suggestion
    const { name: club_name } = await getClub(club_uuid);
```